ci/playbooks/edpm/update.yml

```diff
@@ -33,4 +33,8 @@
           {%- endif %}
           -e update_playbook_run=true
           -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml"
-          -e "cifmw_repo_setup_promotion={{ cifmw_repo_setup_promotion_target }}"
+          {%- if cifmw_update_extras is defined %}
```

Comment:
@sathlan o/ hello. maybe you can join tomorrow's ci reviews call if you have time @ 1100 utc (will add you to the invite momentarily).

For example even this cifmw_repo_setup_promotion is a very specific var which was added (I assume?) for the minor update, so most of us don't even know about that. 

It would be good to get some more context/5 minutes explanation for the team, not just for this review but so we can at least start to hear about the update process and what it will look like in the framework and ci generally. 

sincerely, 

marios

Reply:
Could not join, sorry about that.  I'm a bit confused by that comment though. `cifmw_repo_setup_promotion` is a standard `repo_setup` variable (https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/repo_setup/defaults/main.yml#L23) .  Maybe you meant `cifmw_repo_setup_promotion_target` ?

If so that was the original mechanism used to be able to pass variable to the update playbook run so that we could differentiate them from deployment variables.

Here the review want to get rid of those explicit extra variables and use `cifmw_update_extras` as a list of hashes that will be expanded for the update playbook run.

Then in the job we can define things like this:

```yaml
cifmw_update_extras:
  - cifmw_repo_setup_promotion: latest
```

Does it clarify the matter ?

Reply:
In example
`cifmw_update_extras:
  - cifmw_repo_setup_promotion: latest
` should be:
`cifmw_update_extras:
  cifmw_repo_setup_promotion: latest
` as cifmw_update_extras is dict, not a list

Reply:
thanks for explanation folks marking resolved

primarily my concerns were who is going to be affected by removal of 

```
-e "cifmw_repo_setup_promotion={{ cifmw_repo_setup_promotion_target }}"

```

and confirmed by Sofer just now in slack, this update.yml is only used for the minor update job which you are working on